docs/logic-simple-expr.md

```diff
@@ -60,7 +60,7 @@ Conditional expressions can be used to perform different computations depending
 
 > Typing `return if 1.0 < 0.0 then "negative" else "positive"` in the [Ergo REPL](https://ergorepl.netlify.com), should answer `Response. "positive" : String`.
 
-See also the [Conditional Expression Reference](ergo-reference.html#condition-expressions)
+See also the [Conditional Expression Reference](ref-logic.html#condition-expressions)
```

Comment:
This should be `ref-ergo-spec.html#condition-expressions`

Reply:
Fixed in 9221b7ce9327dad1b1e97cb99ed3608b44baa22c